search/search_repository.go

```diff
@@ -373,6 +373,8 @@ func (r *GormSearchRepository) SearchFullText(ctx context.Context, rawSearchStri
 func init() {
 	// While registering URLs do not include protocol becasue it will be removed before scanning starts
 	// Please do not include trailing slashes becasue it will be removed before scanning starts
-	RegisterAsKnownURL("work-item-details", `(?P<domain>demo.almighty.io)(?P<path>/work-item-list/detail/)(?P<id>\d*)`)
-	RegisterAsKnownURL("localhost-work-item-details", `(?P<domain>localhost)(?P<port>:\d+){0,1}(?P<path>/work-item-list/detail/)(?P<id>\d*)`)
+	RegisterAsKnownURL("work-item-details", `(?P<domain>demo.almighty.io)(?P<path>/work-item/list/detail/)(?P<id>\d*)`)
```

Comment:
This `RegisterAsKnownURL` is in `init()` so I am unable to access `RequestData.Host`with this flow. Yes, I can do these funciton calls in controller on every request and make function look like this ->`RegisterIfNotAlready`.
But as of now, I am trying to make use of - `configuration.GetHTTPAddress()` in current flow of code.